custom/speech.py

```diff
@@ -0,0 +1,248 @@
+from plugin import plugin, require, LINUX
+from colorama import Fore
+import Jarvis
+
+
+speech_installed = True
+try:
+    import speech_recognition as sr  
+    import playsound
+    import gtts
+    #from gtts import gTTS 
+    import os
+    import wolframalpha
+    from selenium import webdriver
+    pass
+except ImportError:
+    print("import error")
+    speech_installed = False
+
+if speech_installed:
+    requirements = []
+else:
+    requirements = [
+        'speech_installed (install speech_recognition playsound + re-run setup.sh)']
+
+
+#global values
+num = 1
+
+
+@require(native=requirements, platform=LINUX)
+@plugin("open speech")
+def open_speech__LINUX(jarvis, s):
```

Comment:
how does this compare to ``jarvicscli/plugins/voice_control.py``?

(And yes: voice_control.py is still in a really early stage, I have multiple ideas to improve but didn't spent much time implementing yet ;) )

Reply:
hello, this is the first time that I try to contribute to an open project with many people, sorry about my mistakes ! I'm doing this plugin to satisfy the requirements of my teacher, and, I tried "hear"  plugin, it did work with me (I guess the problem is caused by the driver of SoundCard), I can't solve with this problem and I don't really understand voice_control.py.  I made this plugin "speech", it is an independent plugin like the other plugins, it can recognize human voice quite well. I don't know what I'm saying, maybe you have already found some unperfects about voice control, I'm finishing speech in quite tight time without understanding all the code of Jarvis, emmmh, I don't know what is going to do........maybe you can give me some advice